app/src/main/kotlin/com/hippo/ehviewer/client/EhEngine.kt

```diff
@@ -446,7 +445,10 @@ object EhEngine {
     }
 
     suspend fun addFavorites(galleryList: List<Pair<Long, String>>, dstCat: Int) {
-        galleryList.parMap(concurrency = Settings.multiThreadDownload) { (gid, token) ->
+        galleryList.forEachIndexed { index, (gid, token) ->
```

Comment:
Do we actually need to sequentialize this?

Reply:
Yeah I was thinking.
Delay on every request feels too much.

Reply:
I mean, anyone really being IP banned due to this?

Reply:
If someone has like 1k local favorites and tries moving them to cloud, he can easily get banned.